packages/terra-date-picker/tests/jest/DateInput.test.jsx

```diff
@@ -19,6 +19,7 @@ it('should render a default date input with all props', () => {
       onChange={() => {}}
       onClick={() => {}}
       onKeyDown={() => {}}
+      aria-label="date"
```

Comment:
How come this was added? To test that custom `aria-label`s cannot be used? Is that the behavior we want?

Reply:
Removed in 009e0a3